x-pack/legacy/plugins/siem/public/pages/hosts/index.tsx

```diff
@@ -224,8 +217,8 @@ export const HostsContainer = pure<HostComponentProps>(({ match }) => (
           )}
         />
         <Redirect
-          from={`${match.url}/:detailName`}
-          to={`${match.url}/:detailName/${HostsTableType.authentications}`}
+          from={`${url}/:detailName`}
+          to={`${url}/:detailName/${HostsTableType.authentications}`}
```

Comment:
Replaced the `match` with just the string `url` so this React.memo won't re-render its self continuously along with its children since the React `match` will be created anew all the time.